apps/PiKISS GUI/install

```diff
@@ -3,6 +3,11 @@
 git_clone https://github.com/Jai-JAP/pikiss-gui || error 'Failed to clone PiKISS GUI repository!'
 
 install_packages ninja-build gobject-introspection libgee-0.8-dev libgirepository1.0-dev libgtk-3-dev valac libgnome-menu-3-dev || exit 1
+
+if package_installed meson; then
+  sudo apt-get purge -y meson || error 'Failed to uninstall older version of meson'
```

Comment:
This could be dangerous, as other packages or apps from P-Apps may depend on the `meson` package. Purging meson like this will also purge anything that requires it.

Reply:
Ok then we have to just force update meson from pip

Reply:
> Ok then we have to just force update meson from pip

I think that would be a lot safer.